analytics/src/main/java/com/segment/analytics/Analytics.java

```diff
@@ -585,6 +589,45 @@ public void run() {
         });
   }
 
+  public void attemptGoal(
+          final @NonNull String event,
+          final @Nullable Properties properties,
+          @Nullable final Options options,
+          @NonNull final AttemptGoalPayload.YesCallback yesCallback,
+          @Nullable final AttemptGoalPayload.NoCallback noCallback) {
+    assertNotShutdown();
+    if (isNullOrEmpty(event)) {
+      throw new IllegalArgumentException("event must not be null or empty.");
+    }
+
+    analyticsExecutor.submit(
+            new Runnable() {
+              @Override
+              public void run() {
+
+                final Options finalOptions;
+                if (options == null) {
+                  finalOptions = defaultOptions;
+                } else {
+                  finalOptions = options;
+                }
```

Comment:
This could be
  final Options finalOptions = options == null ? defaultOptions : options;

Reply:
followed existing example